doc/reference_coverage.xml

```diff
@@ -0,0 +1,195 @@
+      <para>If polygons are found that are not clean relative to their neighbors, the problematic edges will be returned as MULTILINESTRING. For clean polygons the return value will be a NULL. Any non-polygonal or empty geometries on the input will be generate NULL returns.</para>
+      <para>Availability: 3.4.0 - requires GEOS &gt;= 3.12.0</para>
+    </refsection>
+
+    <refsection>
+      <title>Examples</title>
+
+      <programlisting>CREATE TABLE coverage (id integer, geom geometry);
+
+INSERT INTO coverage VALUES
+  (1, 'POLYGON((0 0, 10 0, 10.1 5, 10 10, 0 10, 0 0))'),
+  (2, 'POLYGON((10 0, 20 0, 20 10, 10 10, 10.05 5, 10 0))'),
+  (3, 'POLYGON((20 0, 30 0, 30 10, 20 10, 20 0))');
+
+SELECT id, ST_AsText(ST_CoverageInvalidLocations(geom) OVER ())
+  FROM coverage;
+
+ id |           st_astext
+----+--------------------------------
+  1 | LINESTRING(10 0,10.1 5,10 10)
+  2 | LINESTRING(10 10,10.05 5,10 0)
+  3 |
+      </programlisting>
+
+      <programlisting>-- Test entire table for validity
+SELECT true = ALL (
+    SELECT ST_CoverageInvalidLocations(geom) OVER () IS NULL
+    FROM coverage
+    );
+      </programlisting>
+    </refsection>
+
+    <refsection>
+      <title>See Also</title>
+      <para>
+        <xref linkend="ST_CoverageUnion" />,
+        <xref linkend="ST_CoverageSimplify" />
+      </para>
+    </refsection>
+
+  </refentry>
+
+
+  <refentry id="ST_CoverageSimplify">
+    <refnamediv>
+      <refname>ST_CoverageSimplify</refname>
+
+      <refpurpose>Window function that simplifies the edges of an implicit coverage formed by a set of polygons, applying the Visvalingam–Whyatt algorithm to the edges.</refpurpose>
+    </refnamediv>
+
+    <refsynopsisdiv>
+      <funcsynopsis>
+        <funcprototype>
+          <funcdef>geometry <function>ST_CoverageSimplify</function></funcdef>
+          <paramdef><type>geometry winset </type>
+            <parameter>geom</parameter></paramdef>
+          <paramdef><type>float8 </type>
+            <parameter>tolerance</parameter></paramdef>
+          <paramdef choice="opt"><type>integer </type>
+            <parameter>simplifyBoundary=0</parameter></paramdef>
+        </funcprototype>
+      </funcsynopsis>
+    </refsynopsisdiv>
+
+    <refsection>
+      <title>Description</title>
+
+      <para>Collects all the polygons in the window partition, and forms a polgyonal coverage by looking for shared edges between the polygons. Once coverage is formed, the shared edges are simplified using the <ulink url="https://en.wikipedia.org/wiki/Visvalingam%E2%80%93Whyatt_algorithm">Visvalingam–Whyatt algorithm</ulink>. For clean polygonal coverage inputs, this will result in a simplification that is consistent between polygons, and retains the coverage property for the outputs.</para>
```

Comment:
```suggestion
      <para>Simplifies shared edges of a polygonal coverage using the <ulink url="https://en.wikipedia.org/wiki/Visvalingam%E2%80%93Whyatt_algorithm">Visvalingam–Whyatt algorithm</ulink>. For clean polygonal coverage inputs, this will result in a simplification that is consistent between polygons, and retains the coverage property for the outputs.</para>
```

The description of what GEOS is doing internally introduces a distinction between an "implicit coverage" and a "polygonal coverage" that could be confusing and is not relevant to the operation of the PostGIS function, IMO.